README.md

```diff
@@ -14,6 +13,7 @@ WinDBG is my favourite debugging and reverse engineering tool of choice for my r
 
  - George Bișoc (GeoB99 - Owner)
  - Joann Mõndresku (Cernodile)
+ - Sigwait
```

Comment:
Only add this if it's okay with @GeoB99. I'd classify rather as contributor than maintainer as that would imply you to have push privileges to this repository.

Reply:
I am okay with this being on a separate headline such as "Notable Contributors" for example or similar. "Maintainers" is exclusively to Joann and I.